contrib/scripts/start-gvproxy.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: The vfkit authors
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script can be used to start a raw disk image with vfkit using gvproxy
+# for usermode networking.
+# The mac address must be 5a:94:ef:e4:0c:ee as this is the address expected by gvproxy.
+#
+# After the VM is running, its ssh port is reachable on port 2223 on localhost (127.0.0.1).
+#
+# If the path to `--listen-vfkit` is too long (more than ~100 characters), then
+# gvproxy/vfkit will fail to start as a unix socket filename must be less than
+# that.
+#
+# This script creates an overlay for the disk image, the disk image is not modified.
+#
+set -exuo pipefail
+
+: "${GVPROXY:=gvproxy}"
+: "${VFKIT:=./out/vfkit}"
+
+DISK_IMAGE="${1?Usage: $0 diskimage}"
+VM_NAME="$(basename ${DISK_IMAGE})"
+
+${GVPROXY} -mtu 1500 -ssh-port 2223 -listen-vfkit unixgram://$(pwd)/${VM_NAME}.sock  -log-file ${VM_NAME}.gvproxy.log --pid-file ${VM_NAME}.gvproxy.pid &
```

Comment:
This command mixes single dash `-log-file` and double dash `--pid-file` even if it works, we should use the same style for consistency.